great_expectations_cloud/agent/cli.py

```diff
@@ -70,9 +78,35 @@ def _parse_args() -> Arguments:
     )
 
 
+def load_dotenv(env_file: pathlib.Path) -> set[str]:
+    """
+    Load environment variables from a file.
+
+    Returns a set of the environment variables that were loaded.
+    """
+    import os
+
+    # throw error if file does not exist
+    initial_vars: set[str] = set(os.environ.keys())  # noqa: TID251 # we aren't actually using these vars
```

Comment:
I'd like to avoid loading all the user's env vars. Can the following be used instead:

```
from dotenv import dotenv_values

config = dotenv_values(".env")
```
"returns a dict with the values parsed from the .env file."

Reply:
Why? 
Users have to opt-in to loading their enviroment variables in 2 different ways.
1. By creating a `env` file.
2. By explicitly passing that file to `gx-agent --env-file`

We are never directly accessing any env variable values in the agent code.

Reply:
Is the idea that we would selectively load environment variables from the `env` file?
That would be a major departure from how `.env` file loading typically works.